WordPress/Classes/ViewRelated/Blog/Onboarding Questions Prompt/OnboardingQuestionsPromptViewController.swift

```diff
@@ -0,0 +1,185 @@
+
+    private func configureButtons() {
+        [statsButton, postsButton, notificationsButton, readButton, notSureButton].forEach {
+            style(button: $0)
+        }
+    }
+
+    private func style(button: UIButton) {
+        button.titleLabel?.font = WPStyleGuide.fontForTextStyle(.headline)
+        button.setTitleColor(.text, for: .normal)
+        button.titleLabel?.textAlignment = .natural
+        button.titleEdgeInsets.left = 10
+        button.imageView?.contentMode = .scaleAspectFit
+        button.flipInsetsForRightToLeftLayoutDirection()
+    }
+}
+
+// MARK: - UINavigation Controller Delegate
+extension OnboardingQuestionsPromptViewController {
+    func navigationControllerSupportedInterfaceOrientations(_ navigationController: UINavigationController) -> UIInterfaceOrientationMask {
+        return supportedInterfaceOrientations
+    }
+
+    func navigationControllerPreferredInterfaceOrientationForPresentation(_ navigationController: UINavigationController) -> UIInterfaceOrientation {
+        return .portrait
+    }
+}
+
+
+// MARK: - CGSize Helper Extension
+private extension CGSize {
+
+    /// Get the center point of the size in the given rect
+    /// - Parameter rect: The rect to center the size in
+    /// - Returns: The center point
+    func centered(in rect: CGRect) -> CGPoint {
+        let x = rect.midX - (self.width * 0.5)
+        let y = rect.midY - (self.height * 0.5)
+
+        return CGPoint(x: x, y: y)
+    }
+}
+
+// MARK: - Emoji Drawing Helper Extension
+private extension String {
+    func image() -> UIImage {
+        let size = Constants.iconSize
+        let imageSize = CGSize(width: size, height: size)
+        let rect = CGRect(origin: .zero, size: imageSize)
+
+        UIGraphicsBeginImageContextWithOptions(imageSize, false, 0)
+
+        UIColor.clear.set()
+        UIRectFill(rect)
+
+        let attributes: [NSAttributedString.Key: Any] = [.font: UIFont.systemFont(ofSize: size)]
+
+        let string = self as NSString
+        let drawingSize = string.size(withAttributes: attributes)
+        string.draw(at: drawingSize.centered(in: rect), withAttributes: attributes)
+
+        let image = UIGraphicsGetImageFromCurrentImageContext()
+        UIGraphicsEndImageContext()
+
+        return image?.withRenderingMode(.alwaysOriginal) ?? UIImage()
+    }
+}
+
+// MARK: - Helper Structs
+private struct Strings {
+    static let stats = NSLocalizedString("Checking stats", comment: "Title of a button")
+    static let writing = NSLocalizedString("Writing blog posts", comment: "Title of a button")
+    static let notifications = NSLocalizedString("Staying up to date with notifications", comment: "Title of a button")
+    static let reader = NSLocalizedString("Reading posts from other sites", comment: "Title of a button")
+    static let notSure = NSLocalizedString("Not sure, show me around", comment: "Title of a button")
+    static let skip = NSLocalizedString("Skip", comment: "Title of a button")
```

Comment:
[nit] What do you think of adding a more descriptive comment for these localized strings?

Reply:
Done in: f4c43f4f8bcefe87c4add645a9178e9b0f9483af